### PR TITLE
feat (reporters.json) Add additional Tax Reporters

### PR DESCRIPTION
**Pull request by flooie**
Update to reporters-db

Added two variations on T.C. 
Added T.C. Memo
Added T.C. Summary Opinions
Added T.C. No.

### CODE REVIEW

File: reporters_db/data/reporters.json [hunk 1]
```diff
@@ -8466,6 +8469,64 @@
...
+                "T.C. No.": {
+                    "end": null,
+                    "start": "1942-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:c;tax.court"
+            ],
+            "notes": "T.C. No. is the neutral citation for the T.C. Reporter",
+            "name": "Tax Court Opinions -- that are precedential",
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: I'd suggest this making this `Tax Court Opinions`. We don't need it to note that it's different beyond the `notes` field.

**Comment by mlissner** [comment]
This looks good to me aside from the one comment. The tests are failing on python 3, which is worth fixing too, if you don't mind. I'm guessing it'd be easy. Do you mind taking a look and fixing it if so?

File: reporters_db/data/reporters.json (hunk 1 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: updated

File: reporters_db/data/reporters.json [hunk 2]
```diff
@@ -86,21 +86,25 @@
...
+                "us:c;tax.court"
+            ],
             "name": "American Federal Tax Reports",
             "variations": {
                 "A.F.T.R.": "A.F.T.R. (RIA)",
                 "A.F.T.R. 2d (RIA)": "A.F.T.R.2d (RIA)",
                 "A.F.T.R. 2d": "A.F.T.R.2d (RIA)",
-                "A.F.T.R.2d": "A.F.T.R.2d (RIA)"
+                "A.F.T.R.2d": "A.F.T.R.2d (RIA)",
+                "P-H T.C. Memo.": "P-H T.C. Memo."
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: The right sides of these should be values from the `editions` key. but `P=H T.C. Mem.` isn't. I'm surprised this isn't breaking a test. If it's not, it should be — perhaps we should make a quick test for it.

File: reporters_db/data/reporters.json [hunk 3]
```diff
@@ -8466,6 +8475,64 @@
...
+                "T.C. Summary Opinion": {
+                    "end": null,
+                    "start": "1942-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:c;tax.court"
+            ],
+            "name": "Tax Court Summary Opinions",
+            "notes": "Neutral Citation for summary opinions of the Tax Court. These opinions are atypical format -> T.C. Summary Opinion YYYY-XX",
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: This is great, thanks.

File: reporters_db/data/reporters.json [hunk 4]
```diff
@@ -8466,6 +8475,64 @@
...
+            "editions": {
+                "T.C. No.": {
+                    "end": null,
+                    "start": "1942-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:c;tax.court"
+            ],
+            "notes": "T.C. No. is the neutral citation for the T.C. Reporter",
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: Are these precedential? If so, perhaps it makes sense to mention that as a contract to `T.C. Memo`?

File: tests.py [hunk 5]
```diff
@@ -167,6 +167,14 @@ def test_nothing_ends_before_it_starts(self):
                                 "starts." % k
                         )
 
+    def test_reporter_exists(self):
```
Comment (mlissner) [inline tests.py]: This just checks that the `T.C.` reporters exist? Do we need this? If so, why not add it for `F.` as well?

File: reporters_db/data/reporters.json [hunk 6]
```diff
@@ -103,8 +103,7 @@
                 "A.F.T.R.": "A.F.T.R. (RIA)",
                 "A.F.T.R. 2d (RIA)": "A.F.T.R.2d (RIA)",
                 "A.F.T.R. 2d": "A.F.T.R.2d (RIA)",
-                "A.F.T.R.2d": "A.F.T.R.2d (RIA)",
-                "P-H T.C. Memo.": "P-H T.C. Memo."
+                "A.F.T.R.2d": "A.F.T.R.2d (RIA)"
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: Shouldn't this be: `"P-H T.C. Memo.": "A.F.T.R. (RIA)",`. Confused by it disappearing.

File: reporters_db/data/reporters.json [hunk 7]
```diff
@@ -1597,10 +1595,7 @@
             },
             "mlz_jurisdiction": [],
             "name": "Neutral citation for Court of International Trade",
-            "variations": {},
```
Comment (mlissner) [inline reporters_db/data/reporters.json]: I'm confused. Most of the other entries don't need empty fields?

File: reporters_db/data/reporters.json (hunk 2 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: The test actually was crashing for the it . 

 I fixed the Ph.T.C. Memo variation.

File: reporters_db/data/reporters.json (hunk 4 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: I do not think they are precedential

File: tests.py (hunk 5 above)
Comment (flooie) [reply tests.py]: No- I thought this was removed from the PR.  It is now removed.

File: reporters_db/data/reporters.json (hunk 6 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: You are correct, this has been fixed.

File: reporters_db/data/reporters.json (hunk 7 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: I did not add this variations empty value.  But it was not breaking any tests so I did not remove it.

File: reporters_db/data/reporters.json (hunk 3 above)
Comment (flooie) [reply reporters_db/data/reporters.json]: √

**Pull request merged**